internal/drand-cli/dkg_cli.go

```diff
@@ -403,7 +438,14 @@ func rejectDKG(c *cli.Context) error {
 
 func abortDKG(c *cli.Context) error {
 	err := runSimpleAction(c, func(beaconID string, client drand.DKGControlClient) error {
-		_, err := client.StartAbort(context.Background(), &drand.AbortOptions{BeaconID: beaconID})
+		_, err := client.Command(c.Context, &drand.DKGCommand{
+			Command: &drand.DKGCommand_Abort{Abort: &drand.AbortOptions{
+				BeaconID: beaconID,
+			}},
+			Metadata: &drand.CommandMetadata{
+				BeaconID: beaconID,
```

Comment:
Why can't we use the metadata to provide the `beaconID` and need the redundant data in the packets? 
Same question for all the above using the same pattern.

Reply:
I was concerned that two differently-typed, empty protobuf packets might be the same over the wire, but it seems from my research that is not the case. Will change it

Reply:
okay, have made this change in a single commit for easy reviewing